pycbc/io/hdf.py

```diff
@@ -693,13 +694,16 @@ def get_column(self, cname):
 
 
 class ForegroundTriggers(object):
-    # FIXME: A lot of this is hardcoded to expect two ifos
+
+    # Injection files are expected to only have 'exclusive' IFAR/FAP values,
+    # should use has_inc=False for these.
     def __init__(self, coinc_file, bank_file, sngl_files=None, n_loudest=None,
-                     group='foreground'):
+                 group='foreground', has_inc=True):
         self.coinc_file = FileData(coinc_file, group=group)
         if 'ifos' in self.coinc_file.h5file.attrs:
             self.ifos = self.coinc_file.h5file.attrs['ifos'].split(' ')
         else:
+            # Legacy behaviour for old 2-ifo coinc pipeline
```

Comment:
Just remove the legacy code, this would have been a place I missed it, as it wasn't always easy to find!